src/app/services/interceptor/api.interceptor.ts

```diff
@@ -14,19 +14,22 @@ export class ApiInterceptor implements HttpInterceptor {
     let authReq = req;
 
     if (token) {
-      if (req.url.includes('/logout')) {
         authReq = req.clone({
           setHeaders: {
-            'X-auth-token': `bearer ${token}`
+            'X-auth-token': token
           }
         });
+<<<<<<< HEAD
+
+=======
       } else {
         authReq = req.clone({
           setHeaders: {
             'X-auth-token': token
           }
         });
       }
+>>>>>>> 5ced4070db8b02e61066dc72d4969080a7e87559
```

Comment:
commit head shashum is present and that mean the merge conflict is not resolved properly & the UI should be tested before raising the PR